contrib/generate-gresource-xml.py

```diff
@@ -23,6 +23,6 @@
         n_file.set("preprocess", "xml-stripblanks")
     n_file.set("alias", os.path.basename(fn))
 with open(sys.argv[1], "wb") as f:
-    f.write(ET.tostring(root, "utf-8", xml_declaration=True))
+    f.write(ET.tostring(root, "utf-8"))
```

Comment:
I think I'd rather have:

    try:
        f.write(ET.tostring(root, "utf-8", xml_declaration=True))
    except TypeError:
        f.write(ET.tostring(root, "utf-8"))

I think the `xml_declaration` is important.